frontend/src/components/Loader.js

```diff
@@ -3,18 +3,19 @@ import PropTypes from 'prop-types';
 
 import './Loader.css';
 
-function Loader({ loading }) {
+function Loader({ loading, loadingLabel }) {
   return (
     <>
       {loading && (
-        <div role="status" className="overlay" aria-label="loading" />
+        <div role="status" aria-live="polite" className="overlay" aria-label="loading" loadingLabel={loadingLabel} />
```

Comment:
Feels like loadingLabel was supposed to be used as the value for aria-label here?

Reply:
Oops! yeah good catch!